### PR TITLE
Further changes to AAP on GCP doc

### PR DESCRIPTION
**Pull request by ianf77**
A lot of changes

### CODE REVIEW

File: stories/assembly-aap-gcp-automation-workloads.adoc [hunk 1]
```diff
@@ -6,10 +6,10 @@ ifdef::context[:parent-context: {context}]
 
 = Automation workloads
 
-{PlatformNameShort} from {GCPMarket} is designed to automate the number of managed nodes included with the offer identified at purchase time. 
+{AAPonGCP} is designed to automate the number of managed nodes included with the offer identified at purchase time. 
 This is typically 100 nodes. 
 
-Consult your GCP offer used when deploying {PlatformNameShort} from {GCPMarket}. 
+Consult your GCP offer used when deploying {AAPonGCP}.
```
Comment (ariordan-redhat) [inline stories/assembly-aap-gcp-automation-workloads.adoc:12]:
```suggestion
Refer to your GCP offer used when deploying {AAPonGCP}. 
```

File: stories/assembly-aap-gcp-intro.adoc [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html-single/managing_containers_in_private_automation_hub/index[{PrivateHubNameStart}]
+* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.2/html/getting_started_with_automation_services_catalog/index[{CatalogNameStart}]
+* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.2/html-single/publishing_proprietary_content_collections_in_automation_hub/index[Ansible Content Collections]
+* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.2/html-single/creating_and_consuming_execution_environments/index[{ExecEnvNameStart}]
+* link:https://www.ansible.com/products/automation-platform[Ansible content tools, including access to {InsightsName}]
 
 [NOTE]
 =====
-{AutomationMeshStart} is not available on {PlatformNameShort} from {GCPMarket} at this time and is planned for a later release.
+{AutomationMeshStart} is not available on {AAPonGCP} at this time and is planned for a later release.
```
Comment (ariordan-redhat) [inline stories/assembly-aap-gcp-intro.adoc:23]:
```suggestion
{AutomationMeshStart} is not available on {AAPonGCP} at this time.
```
Suggestion: don't mention the future plans, or customers will plague support, asking when it'll be included.

File: stories/topics/con-aap-gcp-required-apis.adoc [hunk 3]
```diff
@@ -2,8 +2,9 @@
 
 = Required APIs
 
-Your GCP project requires access to several API services to complete the {PlatformNameShort} installation. 
-On deployment, the {PlatformNameShort} installation automatically attempts to enable the followwing APIs. 
+Your {GCP} (GCP) project requires access to several API services to complete {PlatformNameShort} installation. 
+On marketplace deployment, the process automatically attempts to enable the followwing APIs.
```
Comment (ariordan-redhat) [inline stories/topics/con-aap-gcp-required-apis.adoc:6]:
```suggestion
On marketplace deployment, the process automatically attempts to enable the following APIs. 
```

File: stories/topics/proc-aap-gcp-application-deploy.adoc [hunk 4]
```diff
@@ -2,13 +2,13 @@
 
 = Application deployment
 
-Now that you have subscribed to an offer, you can begin configuration before launching the application.
+Now that you have launched the public or private offer, you can begin configuration before deploying the application.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-gcp-application-deploy.adoc:5]: Do we need to say "public or private" ?

File: stories/topics/proc-aap-gcp-application-deploy.adoc [hunk 5]
```diff
@@ -17,32 +17,35 @@ The calculation below will help you find the maximum length of the name of an {P
...
-In the *Capabilities section*, check to acknowledge that CloudFormation may create IAM resources.  
-. Click btn:[Create stack].
+.Procedure
+. In the *Deployment* page, click the btn:[Service Usage API] link below the *Confirm Service Usage API is enabled* checkbox.
+. In the *API/Service Details* tab, ensure that the API is enabled, then return to the deployment page.
+. Check the *Confirm Service Usage API is enabled* checkbox.
+. Select or create a *Service Account*. TBD
+. In the *Region* field, select the region where you want the application deployed.
+. In the *Zone* field, select the zone where you want the Filestore deployed.
+The zone must be in the *Region* you selected before.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-gcp-application-deploy.adoc:27]:
```suggestion
The zone must be in the *Region* you selected.
```

File: stories/topics/proc-aap-gcp-application-deploy.adoc [hunk 6]
```diff
@@ -17,32 +17,35 @@ The calculation below will help you find the maximum length of the name of an {P
...
+[NOTE]
+====
+You will see a warning on the deployment
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+This deployment has resources from the Runtime Configurator service, which is in Beta. 
+----
+
+This warning is expected and not a cause for concern.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-gcp-application-deploy.adoc:49]:
```suggestion
This warning is expected and is not a cause for concern. 
```

File: stories/topics/proc-aap-gcp-secure-communication-SSL.adoc [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+[NOTE]
+====
+Securing traffic through these internal load balancers is different than securing traffic through external load balancers in previous steps. 
+This process ensures HTTP traffic is encrypted even when the traffic is localized to private GCP VPCs.
+====
+
+. In the GCP console, navigate to the link:https://console.cloud.google.com/net-services/loadbalancing/list/loadBalancers[*Load Balancing*] page.
+. In the search bar, enter the name of your stack to filter down to your load balancers.
+We will modify the internal load balancers, which have the suffix int-lb
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-gcp-secure-communication-SSL.adoc:16]:
```suggestion
To modify the internal load balancers, which have the suffix int-lb
```

File: stories/topics/proc-aap-gcp-secure-communication-SSL.adoc [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+For both the controller and hub load balancers, do the following:
+. For the {ControllerName} load balancer, which has the format `<DEPLOYMENT_NAME>-aap<cntrlr>-int-lb`
+. Click btn:[Edit]. 
+. Click btn:[Frontend configuration]
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-gcp-secure-communication-SSL.adoc:20]:
```suggestion
. Click btn:[Frontend configuration].
```

File: stories/topics/proc-aap-gcp-secure-communication-SSL.adoc [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+. Click btn:[ADD FRONTEND IP AND PORT].
+Use the following values:
+.. *Protocol*: HTTPS (includes HTTP/2)
+.. *Subnetwork*: Select the available aap-subnet
+.. *Port*: 443
+.. *IP Address*: `<DEPLOYMENT_NAME>-aap<cntrlr/hub>-intl-lb-ip`
+. .If you have already added your certificate, select it.
+.. If you have not added your certificate, click btn:[CREATE A NEW CERTIFICATE]
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-gcp-secure-communication-SSL.adoc:28]:
```suggestion
.. If you have not added your certificate, click btn:[CREATE A NEW CERTIFICATE].
```

File: stories/topics/proc-aap-gcp-secure-communication-SSL.adoc [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+.. Provide a name for your certificate
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-gcp-secure-communication-SSL.adoc:29]:
```suggestion
.. Provide a name for your certificate.
```

File: stories/topics/proc-aap-gcp-secure-communication-SSL.adoc [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+.. Using your previously generated certificate, copy its contents and paste it under *Certificate*.
+.. Using your previously generated certificate key, copy its contents and paste it under *Private Key*.
+. Click btn:[Create].
+. Click btn:[Done]
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-gcp-secure-communication-SSL.adoc:33]:
```suggestion
. Click btn:[Done].
```

File: stories/topics/proc-aap-gcp-secure-communication-SSL.adoc [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+.Optional: Delete HTTP Frontend configuration
+. Click btn:[Update], and confirm update
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-gcp-secure-communication-SSL.adoc:35]:
```suggestion
. Click btn:[Update], and confirm update.
```

File: stories/topics/proc-aap-gcp-secure-communication-SSL.adoc [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+. Select *backend service for internal load balancer*
+. Click btn:[Edit]
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-gcp-secure-communication-SSL.adoc:37]:
```suggestion
. Click btn:[Edit].
```

File: stories/topics/ref-aap-gcp-automation-performance.adoc [hunk 14]
```diff
@@ -8,9 +8,14 @@ Automating outside of the boundaries of these criteria is not supported, but can
...
-| Forks per job | -
+| Concurrent Jobs | 10
+| Forks per job | 10
 |====
 
-TBD
-
+[NOTE]
+====
+{AAPonGCP} is driven using three m5.large instances, two of which run {PlatformName} {ControllerName} and one which runs {HubName}. The {ControllerName} instances collectively support a standard workload for 100 managed active nodes.
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-gcp-automation-performance.adoc:17]: Do we need to format `m5.large` ?

File: stories/topics/ref-aap-gcp-automation-performance.adoc [hunk 15]
```diff
@@ -8,9 +8,14 @@ Automating outside of the boundaries of these criteria is not supported, but can
...
+| Concurrent Jobs | 10
+| Forks per job | 10
 |====
 
-TBD
-
+[NOTE]
+====
+{AAPonGCP} is driven using three m5.large instances, two of which run {PlatformName} {ControllerName} and one which runs {HubName}. The {ControllerName} instances collectively support a standard workload for 100 managed active nodes. 
+Testing this operating criteria by Red Hat found support for up-to 10 forks each.
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-gcp-automation-performance.adoc:18]: Is the dash in "up-to" necessary?

File: stories/topics/ref-aap-gcp-automation-performance.adoc [hunk 16]
```diff
@@ -8,9 +8,14 @@ Automating outside of the boundaries of these criteria is not supported, but can
...
+| Forks per job | 10
 |====
 
-TBD
-
+[NOTE]
+====
+{AAPonGCP} is driven using three m5.large instances, two of which run {PlatformName} {ControllerName} and one which runs {HubName}. The {ControllerName} instances collectively support a standard workload for 100 managed active nodes. 
+Testing this operating criteria by Red Hat found support for up-to 10 forks each.  
+This operating criteria above has been set and tested using output-intensive, “chatty” workloads that produce 2 messages 7 seconds apart on both {ControllerName} nodes.
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-gcp-automation-performance.adoc:19]:
```suggestion
This operating criteria has been set and tested using output-intensive, “chatty” workloads that produce 2 messages 7 seconds apart on both {ControllerName} nodes. 
```

File: stories/topics/ref-aap-gcp-automation-performance.adoc (hunk 15 above)
Comment (ariordan-redhat) [inline stories/topics/ref-aap-gcp-automation-performance.adoc:18]:
```suggestion
Red Hat has tested the operating criteria for up-to 10 forks each.  
```
Suggesting that this be re-phrased.  My attempt could be misinterpreting the sentence, though.

File: stories/topics/ref-aap-gcp-external-load-balancer.adoc [hunk 17]
```diff
@@ -3,15 +3,16 @@
 = Using an external load balancer
 
 It is possible to expose Ansible {ControllerName} and Ansible {PrivateHubName} to the public internet if you wish users to have access to the platform from any internet connected machine. 
-This is not recommended as a best practice for security reasons, but this implementation can be secured with different GCP tools.  This section describes how to connect a load balancer that faces the public internet, but it does not include steps to harden access through Google’s security products. 
+This is not recommended as a best practice for security reasons, however this implementation can be secured with different GCP tools.  This section describes how to connect a load balancer that faces the public internet, but it does not include steps to harden access through Google’s security products.
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-gcp-external-load-balancer.adoc:6]:
```suggestion
This is not recommended as a best practice for security reasons.
However, this implementation can be secured with different GCP tools.  
This section describes how to connect a load balancer that faces the public internet, but it does not include steps to harden access through Google’s security products. 
```

File: stories/topics/ref-aap-gcp-external-load-balancer.adoc [hunk 18]
```diff
@@ -23,31 +24,33 @@ If you do not see *Network Services*, then select *View All Products*.
...
+.. Click btn:[CREATE].
 This returns you to *Backend configuration* section.
-. Click btn:[OK].
-. Click btn:[CREATE] to create the load balancer for the {ControllerName} UI.  
+.. Click btn:[OK].
+.. Click btn:[CREATE] to create the load balancer for the {ControllerName} UI.  
 This will take a few minutes to complete. 
-A load balancer has now been created. 
-. Configure the DNS record for the domain that you used in your SSL certificate to point to the IP address of the load balancer.  
+.A load balancer has now been created.
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-gcp-external-load-balancer.adoc:52]:
```suggestion
A load balancer has now been created. 
```

File: stories/topics/ref-aap-gcp-external-load-balancer.adoc [hunk 19]
```diff
@@ -23,31 +24,33 @@ If you do not see *Network Services*, then select *View All Products*.
...
 This returns you to *Backend configuration* section.
-. Click btn:[OK].
-. Click btn:[CREATE] to create the load balancer for the {ControllerName} UI.  
+.. Click btn:[OK].
+.. Click btn:[CREATE] to create the load balancer for the {ControllerName} UI.  
 This will take a few minutes to complete. 
-A load balancer has now been created. 
-. Configure the DNS record for the domain that you used in your SSL certificate to point to the IP address of the load balancer.  
+.A load balancer has now been created. 
+.Configure the DNS record for the domain that you used in your SSL certificate to point to the IP address of the load balancer.
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-gcp-external-load-balancer.adoc:53]:
```suggestion
. Configure the DNS record for the domain that you used in your SSL certificate to point to the IP address of the load balancer.  
```
Is this a separate step?

File: stories/topics/ref-aap-gcp-external-load-balancer.adoc [hunk 20]
```diff
@@ -23,31 +24,33 @@ If you do not see *Network Services*, then select *View All Products*.
...
+.. Click btn:[OK].
+.. Click btn:[CREATE] to create the load balancer for the {ControllerName} UI.  
 This will take a few minutes to complete. 
-A load balancer has now been created. 
-. Configure the DNS record for the domain that you used in your SSL certificate to point to the IP address of the load balancer.  
+.A load balancer has now been created. 
+.Configure the DNS record for the domain that you used in your SSL certificate to point to the IP address of the load balancer.  
 
 At this point you should have accesss to the {PlatformNameShort} {Controllername} UI.  
 You can log after you retrieve the admin password.
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-gcp-external-load-balancer.adoc:56]:
```suggestion
You can log in after you retrieve the admin password.
```

File: stories/topics/ref-aap-gcp-networking-options.adoc [hunk 21]
```diff
@@ -0,0 +1,48 @@
...
+
+The deployment creates a new VPC and subnet that are not accessible from the public internet. 
+You must provide access to the application as described in xref:assembly-aap-gcp-networking-access[Networking and Application Access]. 
+
+Consider your existing network configurations when specifying the following network ranges below. 
+Ensure that each range does not overlap with any other specified here, and does not overlap with existing ranges in your network. 
+Each range should exist within a private network class.
+
+*Application Subnet Range*:  Network CIDR defining the subnet range used by the custom VPC deployed by the offering. 
+Must be a minimum “/24” segment and must be in the private network range (192.168 or 10.0).
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-gcp-networking-options.adoc:15]:
```suggestion
Must be a minimum “/24” segment and must be in the private network range (192.168 or 10.0).  CIDR ranges be enclosed in backticks? `/24`
```

File: stories/topics/ref-aap-gcp-security-considerations.adoc [hunk 22]
```diff
@@ -2,4 +2,8 @@
 
 = Security considerations
 
-TBD
+To configure {RHSSO} with an identity provider that can enable MFA, follow the steps link:https://docs.ansible.com/ansible-tower/latest/html/administration/ent_auth.html[here] for connecting enterprise authentication to {PlatformNameShort}.
+
+Securing infrastructure services is an important step to any cloud deployment.
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-gcp-security-considerations.adoc:7]:
```suggestion
Securing infrastructure services is an important step in any cloud deployment. 
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
There's a typo, but the rest of the comments are just suggestions.

File: stories/topics/ref-aap-gcp-automation-performance.adoc (hunk 14 above)
Comment (ianf77) [reply stories/topics/ref-aap-gcp-automation-performance.adoc:17]: I've not seen it formatted before.

**Review by ariordan-redhat — APPROVED** [review]
Looks good to me

**Review by ariordan-redhat — APPROVED** [review]

**Pull request merged**